tools/topology/topology2/cavs-mixin-mixout-ssp.conf

```diff
@@ -0,0 +1,95 @@
+#
+# Pipeline definitions
+#
+# PCM0 ---> gain ----> Mixin ----> Mixout ----> gain ----> SSP0
+# PCM1 ---> gain ----> Mixin ----> Mixout ----> gain ----> SSP1
+#
+
+# Pipeline ID:1 PCM ID: 0
+Object.Pipeline {
+	# playback pipelines
+	host-copier-gain-mixin-playback [
+		{
+			index	1
+
+			Object.Widget.host-copier.1 {
+				stream_name	'$HEADSET_PLAYBACK_PCM_STREAM_NAME'
+				pcm_id $HEADSET_PCM_ID
+			}
+			Object.Widget.gain.1 {
+				Object.Control.mixer.1 {
+					name	'Pre Mixer $HEADSET_PCM_NAME Playback Volume'
+				}
+			}
+		}
+		{
+			index	3
+
+			Object.Widget.host-copier.1 {
+				stream_name	'$SPEAKER_PLAYBACK_PCM_STREAM_NAME'
+				pcm_id $SPEAKER_PCM_ID
+			}
+			Object.Widget.gain.1 {
+				Object.Control.mixer.1 {
+					name	'Pre Mixer $SPEAKER_PCM_NAME Playback Volume'
+				}
+			}
+		}
+	]
+
+        mixout-gain-dai-copier-playback [
```

Comment:
this seems a bit too much duplication. Can we simply not create a new pipeline with dts called mixout-eqiir-dts-playback and then replace this with mixout-gain-dai-copier-playback when needed?

Reply:
I have one question. For "insert" you mentioned, do you mean using IncludeByKey to choose different pipeline or route at the top-level?